example.json

```diff
@@ -0,0 +1,3 @@
+{
+	"linkTargetSitelinkSiteId": "enwiki"
```

Comment:
It seems that this config is still unused. Is that correct?

Reply:
Yes. I'll update this now that https://github.com/ProfessionalWiki/WikibaseFacetedSearch/pull/24 is merged.